src/aap_eda/wsapi/consumers.py

```diff
@@ -575,3 +586,26 @@ def get_vault_password_and_id(
             vault_inputs = yaml.safe_load(vault_inputs.get_secret_value())
             return vault_inputs["vault_password"], vault_inputs["vault_id"]
         return None, None
+
+    @staticmethod
+    def get_file_content_message(
+        template: str, binary_fields: list[str], value: str, inputs: dict
+    ) -> FileContentMessage:
+        binary_file = False
```

Comment:
`binary_file = any(attr in binary_fields for attr in extract_variables(value))` may be better